src/MatLang.jl

```diff
@@ -1,20 +1,76 @@
 module MatLang
 using Requires
+using LinearAlgebra
 
-export clcM
+# Main Functions
+export clcM, eyeM, diagM
+
+# Native Julia functions that are just a copy:
+export zerosM, onesM, randM
+
+
+# Command Window (REPL console):
 ################################################################
 """
     clcM
 
 Clear Command Window for Juno or Atom IDE
 """
 function clcM()
-#    Juno.clearconsole();
+#    Juno.clearconsole(); # defined in @require instead
 end
 ################################################################
 
 
 ################################################################
+
+# Matrices and Arrays:
+################################################################
+const zerosM = zeros
+################################################################
+const onesM = ones
+################################################################
+const randM = rand
+################################################################
+"""
+    eyeM(s1,s2,...)
+    eyeM(sizeAsTuple)
+
+Creates Identity matrix (can be non-square matrix).
+Example
+'''jl
+mEye=eyeM(2,3); # [1 0 0; 0 1 0]
+mEye2=eyeM((2,2))  # giving size as a Tuple. In Julia we use (2,2) instead of giving it as [2,2]
+s1=size(ones(2,3))
```

Comment:
```diff
- s1=size(ones(2,3))
+ s1 = (2, 3)
```

Reply:
I wanted to demonstrate that size can come from another matrix. 
Also, I will create another method that converts a matrix to a tuple, since, In Matlab, we can pass size as a matrix (instead of tuple).